src/Features/Core/Portable/CodeRefactoringHelpers.cs

```diff
@@ -131,5 +166,38 @@ internal static class CodeRefactoringHelpers
 
             return TextSpan.FromBounds(start, end);
         }
+
+        /// <summary>
+        /// Strips leading and trailing whitespace from <paramref name="span"/>.
+        /// </summary>
+        /// <remarks>
+        /// Returns unchanged <paramref name="span"/> in case <see cref="TextSpan.IsEmpty"/>.
+        /// Returns empty Span with original <see cref="TextSpan.Start"/> in case it contains only whitespace.
+        /// </remarks>
+        private static async Task<TextSpan> GetStrippedTextSpan(Document document, TextSpan span, CancellationToken cancellationToken)
+        {
+            var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            var start = span.Start;
+            var end = span.End;
+
+            if (span.IsEmpty)
+            {
+                return span;
+            }
+
+            while (start < end && char.IsWhiteSpace(sourceText[end - 1]))
+            {
+                end--;
+            }
+
+            while (start < end && char.IsWhiteSpace(sourceText[start]))
+            {
+                start++;
+            }
+
+            return start == end
+                ? new TextSpan(start, 0)
+                : TextSpan.FromBounds(start, end);
```

Comment:
no need for this conditional.  just: `return TextSpan.FromBounds(start, end);`

Reply:
Fails, there's check in FromBounds that makes sure it's non empty. TBH don't understand why, but it's there.

Reply:
wat...  seriously.  wtf :)